api/services/diff/v1/diff.proto

```diff
@@ -23,10 +23,11 @@ service Diff {
 }
 
 message ApplyRequest {
+	string snapshotter = 1;
```

Comment:
We don't need this here, there is a PR to add a snapshotter mount type which have this as an option. The way the differ is designed now though, snapshot ids should not be explicit in the API